sdk/src/events/eventSubscriber.ts

```diff
@@ -33,7 +34,7 @@ export class EventSubscriber {
 	public lastSeenTxSig: string;
 
 	public constructor(
-		private connection: Connection,
+		private connection: Hybrid,
```

Comment:
instead of adding Hybrid here and doing ts-ignore around the sdk, i think it'd be nicer to just do ts-ignore in tests